packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md

```diff
@@ -6,7 +6,7 @@ title: Basic sign-in flow with the password factor
 
 Enable a password-only sign-in flow in your web app using the embedded SDK.
 
-> **Note**: Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. Give your users the ability to use other authenticators by replacing password-only sign-in experiences with either a [password-optional](https://developer.okta.com/docs/guides/pwd-optional-overview) or a multifactor experience.
+> **Note**: Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. You can enahnce the security by replacing password-only sign-in experiences with either a [password optional](https://developer.okta.com/docs/guides/pwd-optional-overview) or a multifactor experience.
```

Comment:
```suggestion
> **Note**: Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. You can enhance the security by replacing password-only sign-in experiences with either a [password optional](https://developer.okta.com/docs/guides/pwd-optional-overview) or a multifactor experience.
```